site/reference/pkg/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Pkg"
+title: "`pkg`"
```

Comment:
@etefera Do we actually need this headers? I think they were specific to Hugo which we no longer use. Please follow up in separate PR.

Reply:
Yes, they were here left in order to support compatibility with merging into Hugo branches which isn't necessary anymore. We no longer need those blocks going forward, and it can be removed here in favor of a traditional heading 
`# pkg`.

Reply:
I'll do a separate PR to remove those from the reference docs.